docs/docs/tuning-your-model.mdx

```diff
@@ -434,9 +434,48 @@ underlying tensorflow ops `tf.sparse.sparse_dense_matmul`,
 `tf.nn.sparse_softmax_cross_entropy_with_logits`,
 and `tf.math.unsorted_segment` ops. For more information see [here](https://github.com/tensorflow/community/blob/master/rfcs/20210119-determinism.md)
 
-You can see which featurizers are sparse [here](./components.mdx#featurizers),
+For the above reasons, the models are also not guaranteed to yield the exact same
+performance when trained on a GPU across multiple runs. This even applies to situations where training is run
+multiple times on the same training data with the same config
+and random seeds set appropriately while being evaluated on a standard held-out test set.
+Internal experiments have shown the following fluctuation in model's performance when trained and evaluated on a held-out
+test set of a variety of datasets (experiments were run 5 times on each dataset):
+
+```
++-----------------------+------------+------------------+------------------+-------------------+
+| Task,                 | Number of  | Average standard | Minimum standard | Maximum standard  |
```

Comment:
Can we mention which GPU we used to run this experiment in case people want to replicate this behavior?

Reply:
We don't mention the datasets here on which this study was done because some of them are private customer datasets, so they can't really reproduce the results.

Reply:
I've added the information about the GPU as well though